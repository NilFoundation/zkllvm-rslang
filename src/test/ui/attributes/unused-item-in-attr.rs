#[w = { extern crate alloc; }]
//~^ ERROR unexpected expression: `{
//~| ERROR cannot find attribute `w` in this scope
fn f() {}

fn main() {}
