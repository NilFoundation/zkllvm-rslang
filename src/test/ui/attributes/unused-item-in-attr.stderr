error: unexpected expression: `{
           extern crate alloc;
       }`
  --> $DIR/unused-item-in-attr.rs:1:7
   |
LL | #[w = { extern crate alloc; }]
   |       ^^^^^^^^^^^^^^^^^^^^^^^

error: cannot find attribute `w` in this scope
  --> $DIR/unused-item-in-attr.rs:1:3
   |
LL | #[w = { extern crate alloc; }]
   |   ^

error: aborting due to 2 previous errors

